### PR TITLE
Added Turkish translation

### PR DESCRIPTION
**Pull request by vnumex**

### CODE REVIEW

File: src/locales/index.ts [hunk 1]
```diff
@@ -5,3 +5,4 @@ export { default as fr } from "./fr/translation.json";
 export { default as hu } from "./hu/translation.json";
 export { default as it } from "./it/translation.json";
 export { default as ru } from "./ru/translation.json";
+export { default as tr } from "./tr/translation.json";
```
Comment (hydralauncher) [inline src/locales/index.ts:8]: Newline at the ends missing

File: src/locales/tr/translation.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    "binary_not_found_modal": {
+      "title": "Programlar yüklü değil",
+      "description": "Sisteminizde Wine veya Lutris çalıştırılabiliri bulunamadı",
+      "instructions": "Oyunları düzgün şekilde çalıştırmak için Linux distronuza bunlardan birini nasıl yükleyebileceğinize bakın"
+    },
+    "modal": {
+      "close": "Kapat tuşu"
+    }
+  }
+
```
Comment (hydralauncher) [inline src/locales/tr/translation.json:162]: Newline at the ends missing

**Review by hydralauncher — CHANGES_REQUESTED** [review]

File: src/locales/tr/translation.json (hunk 2 above)
Comment (hydralauncher) [reply src/locales/tr/translation.json:162]: Newline's still missing

**Review by hydralauncher — CHANGES_REQUESTED** [review]

File: src/locales/tr/translation.json (hunk 2 above)
Comment (vnumex) [reply src/locales/tr/translation.json:162]: There's a empty line at the end of the file, what's wrong?

**Review by hydralauncher — APPROVED** [review]
LGTM

**Pull request merged**